examples/functions/python3/mmlogic-simple/harness.py

```diff
@@ -28,13 +29,13 @@
 
 # Step 2 - Talk to Redis.  This example uses the MM Logic API in OM to read/write to/from redis.
 # Establish grpc channel and make the API client stub
-api_conn_info = "%s:%d" % (os.environ["OM_MMLOGICAPI_SERVICE_HOST"],os.environ["OM_MMLOGICAPI_SERVICE_PORT"])
+api_conn_info = "%s:%s" % (os.environ["OM_MMLOGICAPI_SERVICE_HOST"],os.environ["OM_MMLOGICAPI_SERVICE_PORT"])
 with  grpc.insecure_channel(api_conn_info) as channel:
     mmlogic_api = mmlogic_pb2_grpc.MmLogicStub(channel)
 
     # Step 3 - Read the profile written to the Backend API.
     # Get profile from redis
-    profile_pb = mmlogic_api.GetProfile(mmlogic.MatchObject(id=os.environ["MMF_PROFILE_ID"]))
+    profile_pb = mmlogic_api.GetProfile(mmlogic.GetProfileRequest(match = pb.MatchObject(id=os.environ["MMF_PROFILE_ID"]))).match
```

Comment:
Split this into a separate change. Much appreciated fix.

Reply:
done #193